### PR TITLE
Reporting cards on the home page

### PR DESCRIPTION
**Pull request by dadiorchen**
## Description

Screen Shot 2021-12-09 at 5 49 18 PM

### Please check this demo video:
https://dadior.s3.ap-northeast-1.amazonaws.com/Screen%20Recording%202021-12-10%20at%203.26.22%20PM.mp4

**Issue(s) addressed**

- Resolves #_[insert ID of the issue addressed]_

**What kind of change(s) does this PR introduce?**

- [ ] Enhancement
- [ ] Bug fix
- [ ] Refactor

**Please check if the PR fulfills these requirements**

- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## Issue

**What is the current behavior?**

**What is the new behavior?**

## Breaking change

**Does this PR introduce a breaking change?**

### CODE REVIEW

**Comment by dadiorchen** [comment]
@nmcharlton these cards are going to emebed to external website, so I didn't do any permission policy against them.

@ZavenArra to embed the cards, my short-term plan is to open some routers like this: http://admin.treetrackor.org/reporting/card1?start-date=2021-01-01&end-date=2021-12-30 
Just expose one router for one card. 
what do you think?

**Comment by ZavenArra** [comment]
@dadiorchen please fix failed CI issues on this PR

**Comment by ZavenArra** [comment]
> @nmcharlton these cards are going to embed to external website, so I didn't do any permission policy against them.
> 
> @ZavenArra to embed the cards, my short-term plan is to open some routers like this: http://admin.treetrackor.org/reporting/card1?start-date=2021-01-01&end-date=2021-12-30 Just expose one router for one card. what do you think?

Yes, you can open some routes, but we will use a different domain to point to them (same CDN).  reporting.treetracker.org.   We do not want to reference the admin panel in our delivery of these embeds, but it's OK to include them in the same codebase and CDN for deployment.

File: src/components/Routers.js [hunk 1]
```diff
@@ -56,6 +61,36 @@ export default function Routers() {
...
+                      <Route path="/reporting/card4">
+                        <ReportingCard4 {...props} />
+                      </Route>
+                      <Route path="/reporting/card5">
+                        <ReportingCard5 {...props} />
+                      </Route>
+                    </Switch>
+                  );
+                })()}
+              </Route>
```
Comment (dadiorchen) [inline src/components/Routers.js:93]: exposed routers, usage: http://localhost:3001/reporting/card5?start-date=2011-12-22&end-date=2021-02-01

File: src/components/reportingCards/ReportingCard.hook.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      top: data[field1][field2].map((p) => ({
+        name: p.name,
+        num: p.number,
+      })),
+    });
+  }
+
+  React.useEffect(() => {
+    setData(undefined);
+    load('http://47.91.14.192:3006/capture/statistics', startDate, endDate);
```
[the remark this answers is not in the document]
Comment (ZavenArra) [reply src/components/reportingCards/ReportingCard.hook.js]: yes, I think that is good.

File: src/components/Routers.js (hunk 1 above)
Comment (ZavenArra) [reply src/components/Routers.js]: good.

**Comment by ZavenArra** [comment]
Cleaned up build warnings in PR #247 so that I can merge.

**Pull request merged**